### ISSUE TITLE
Translation for Persian/Farsi

### ISSUE THREAD

**Issue by mahdin75** [Project-OSRM/osrm-text-instructions#316]
There's no file for the Persian language.
I can add it to add it to the supported languages.
Can I open a pull request on it?

**Comment by 1ec5** [comment]
Yes, a PR would be welcome. Make sure you pull the translations from Transifex rather than translating the file by hand. It should be based on the `fa` localization, not the `fa-IR` localization. See also #289.
